packages/ruleset/src/rules/integer-attributes.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2017 - 2024 IBM Corporation.
+ * SPDX-License-Identifier: Apache2.0
+ */
+
+const {
+  schemas,
+} = require('@ibm-cloud/openapi-ruleset-utilities/src/collections');
+const { oas3 } = require('@stoplight/spectral-formats');
+const { integerAttributes } = require('../functions');
+
+module.exports = {
+  description: 'Integer schemas should have certain attributes defined',
+  message: '{{error}}',
+  severity: 'warn',
```

Comment:
I might suggest making this rule an `error` in severity. I've been trying to use `error` anytime the Handbook uses "MUST" as it's guidance.

This rule is tricky because the guidance for minimum/maximum is "SHOULD" for integers in responses, but it is "MUST" for requests, as well for the `format` field in both responses and requests. Because most of the checks here have "MUST" applied to them, an `error` might be more appropriate.

Reply:
I sort of flipped a coin on error vs warn on this one b/c of the request vs response discrepancies.   I'll change it to error.

Reply:
fixed